qiskit_experiments/library/characterization/analysis/readout_angle_analysis.py

```diff
@@ -14,31 +14,92 @@
 """
 
 import numpy as np
+from typing import List, Optional
 
-from qiskit_experiments.framework import BaseAnalysis, AnalysisResultData
-
+from qiskit_experiments.framework import BaseAnalysis, AnalysisResultData, Options
+from qiskit_experiments.framework.matplotlib import get_non_gui_ax
 
 class ReadoutAngleAnalysis(BaseAnalysis):
     """
     A class to analyze readout angle experiments
     """
+    
+    @classmethod
+    def _default_options(cls) -> Options:
+        """Return default analysis options.
+
+        Analysis Options:
+            plot (bool): Set ``True`` to create figure for fit result.
+            ax(AxesSubplot): Optional. A matplotlib axis object to draw.
+        """
+        options = super()._default_options()
+        options.plot = True
+        options.ax = None
+        return options
 
-    def _run_analysis(self, experiment_data):
+    def _run_analysis(self, experiment_data, **kwargs):
         angles = []
+        radii = []
+        centers = []
         for i in range(2):
             center = complex(*experiment_data.data(i)["memory"][0])
             angles.append(np.angle(center))
+            radii.append(np.absolute(center))
+            centers.append(center)
 
         angle = (angles[0] + angles[1]) / 2
         if (np.abs(angles[0] - angles[1])) % (2 * np.pi) > np.pi:
             angle += np.pi
 
         analysis_results = [
             AnalysisResultData(
-                name="ReadoutAngle",
-                value=angle,
-                extra={"angle_ground": angles[0], "angle_excited": angles[1]},
-            )
-        ]
+                name="readout_angle_qiskit_monitoring",
```

Comment:
Remove the qiskit-monitrong from the names, we're not specifically in monitoring anymore

Reply:
done.